src/components/CallView/Grid/Grid.vue

```diff
@@ -67,8 +67,6 @@
 									:is-stripe="isStripe"
 									:is-promoted="sharedDatas[callParticipantModel.attributes.peerId].promoted"
 									:is-selected="isSelected(callParticipantModel)"
-									:fit-video="false"
```

Comment:
As you are not passing it as a prop, remove it from Video.

Reply:
It could passed, I just don't see a reason to keep this line here, as fit Video prop is false by default

Reply:
oops, found the other instance of the component. nvm